tc_prometheus/metrics/prometheus_metrics.py

```diff
@@ -87,7 +90,7 @@ def __labels(self, name, metricname):
         values = metricname.replace(name + '.', '').split('.', len(self.mapping[name])-1)
         labels = {}
         for index, label in enumerate(self.mapping[name]):
-            labels[label] = values[index]
```

Comment:
I get the intention here, yet this would mean that we actually have an issue if this happens as we might have a wrong mapping suddenly and so it would go unnotice. In case just a label was dropped, then this would be fine but if a new one was added and the order was changed, we would actually miss this.

I am uncertain therefore if we shouldn't do a proper `raise` in case `len(values) != len(self.mapping[name])` to clearly state the issue. This wouldn't help if someone simply changes the labels (order or content) without changing the amount, but we cannot ensure that one sadly.

WDYT?

Reply:
argh okay I see from the other comments.. there is one which "might" have an extra one. Which is weird and with `network location it is anyways bad and tricky (extra dots).. so hmm.. yeah probably it is fine..